### DISCUSSION TITLE
升级主题后toc失效

### DISCUSSION THREAD

**Discussion by ymkNK** [CaiJimmy/hugo-theme-stack#545] [Q&A]
https://lllovol.com/p/2021/12/refactoring-chapter2/ 升级了最新的主题之后，不知道为什么toc没有生效了，展示不出来了orz
配置文件 https://github.com/ymkNK/hugo/blob/main/config.yaml

**Comment by zhixuan2333** [comment]
请按照 #507 更改你的config

**Reply by ymkNK** [reply]
好的，感谢~确实是配置的问题！万分感谢！！
